### PR TITLE
Fix pixlr image url

### LINKED ISSUE #944: pixlr extension for edit images not found
I configure the pixlr extension but this extension not found.
when I try to edit the following error jumps

### PR DESCRIPTION
**Pull request by onEXHovia**
## Subject

Closes #944 

## Changelog

```markdown
### Fixed
- Fix query parameter `image`. Generate full image URL if possible.
```

### CODE REVIEW

File: src/Extra/Pixlr.php [hunk 1]
```diff
@@ -11,13 +11,15 @@
 
 namespace Sonata\MediaBundle\Extra;
 
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\Pool as PoolAdmin;
```
Comment (OskarStark) [inline src/Extra/Pixlr.php]: Pleas name it AdminPool

File: src/Extra/Pixlr.php [hunk 2]
```diff
@@ -37,7 +39,7 @@ class Pixlr
     protected $secret;
 
     /**
-     * @var MediaManagerInterface
+     * @var ManagerInterface
```
Comment (core23) [inline src/Extra/Pixlr.php]: Why?

Comment (onEXHovia) [reply src/Extra/Pixlr.php]: Currently `Sonata\MediaBundle\Model\MediaManagerInterface` and `Sonata\CoreBundle\Model\ManagerInterface` deprecated. Changes were not needed?

File: src/Extra/Pixlr.php [hunk 3]
```diff
@@ -185,7 +199,7 @@ public function targetAction(Request $request, $hash, $id)
      */
     public function isEditable(MediaInterface $media)
     {
-        if (!$this->container->get('sonata.media.admin.media')->isGranted('EDIT', $media)) {
```
Comment (OskarStark) [inline src/Extra/Pixlr.php]: Please keep the container. Otherwise it is a bc break

And only change the url Part

**Comment by core23** [comment]
Can you please rebase @onEXHovia

**Pull request closed without merging**